include/swift/IDE/IDEBridging.h

```diff
@@ -102,29 +92,40 @@ ResolvedLocVector ResolvedLocVector_createEmpty();
 ///   In that case `swift_SwiftIDEUtilsBridging_runNameMatcher` can return a
 ///   `ResolvedLocVector`.
 class BridgedResolvedLocVector {
-  const std::vector<ResolvedLoc> *vector;
+  friend void *BridgedResolvedLocVector_getOpaqueValue(const BridgedResolvedLocVector &vector);
+
+  std::vector<ResolvedLoc> *vector;
 
 public:
   /// Create heap-allocaed vector with the same elements as `vector`.
   BridgedResolvedLocVector(const std::vector<ResolvedLoc> &vector);
 
   /// Create a `BridgedResolvedLocVector` from an opaque value obtained from
   /// `getOpaqueValue`.
-  BridgedResolvedLocVector(const void *opaqueValue);
+  BridgedResolvedLocVector(void *opaqueValue);
+
+  void push_back(const ResolvedLoc &Loc);
 
   /// Get the underlying vector.
   const std::vector<ResolvedLoc> &unbridged();
 
   /// Delete the heap-allocated memory owned by this object. Accessing
   /// `unbridged` is illegal after calling `destroy`.
   void destroy();
-
-  /// Get an opaque pointer value that describes this 
-  /// `BridgedResolvedLocVector`. This opaque value can be returned by a
-  /// `@_cdecl` function in Swift.
-  const void *getOpaqueValue() const;
 };
 
+
+SWIFT_NAME("BridgedResolvedLocVector.empty()")
+BridgedResolvedLocVector BridgedResolvedLocVector_createEmpty();
```

Comment:
Can this now be a constructor for BridgedResolvedLocVector?

Reply:
I tried it as a constructor and IIRC the importer does not import initializers without parameters in 5.8.

Reply:
Hmm I don't recall having issues for e.g:

https://github.com/apple/swift/blob/d41d9bdc77ebb60a82cafdd02cab6356bc8808cb/include/swift/Basic/BasicBridging.h#L204